packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -109,9 +110,15 @@ export const Dashboard = () => {
   const [exportModalOpen, setExportModalOpen] = useState(false);
   const { data: entity } = useEntity(projectCode, entityCode);
   const bounds = entity?.bounds || DEFAULT_BOUNDS;
+  useEffect(() => {
+    gaEvent('Dashboard', 'Change Tab', activeDashboard?.name);
+  }, [activeDashboard]);
 
   const toggleExpanded = () => {
     setIsExpanded(!isExpanded);
+    if (isExpanded === true) {
+      gaEvent('Pages', 'Toggle Info Panel');
```

Comment:
I'm guessing we want this to be called regardless of the value of `isExpanded`? In the old one I think it's just every time it's expanded